src/Uno.UI/UI/Xaml/Controls/TextBlock/TextBlock.Android.cs

```diff
@@ -227,9 +228,11 @@ private void UpdatePaint()
 				.GetColorWithOpacity(Foreground, Colors.Transparent)
 				.Value;
 
-			var shader = Foreground is GradientBrush gb
-				? gb.GetShader(LayoutSlot.LogicalToPhysicalPixels())
-				: null;
+			var shader = Foreground switch
+			{
+				IGradientBrush gb => gb.GetShader(LayoutSlot.LogicalToPhysicalPixels()),
+				_ => null
+			};
```

Comment:
I love the new `switch` but it's bit overkill here ;)

```suggestion
			var shader = (Foreground as IGradientBrush)?.GetShader(LayoutSlot.LogicalToPhysicalPixels());
```

Reply:
@dr1rrb I initially had more than two arms when I used the `switch`. But that has changed while I was working. Will commit your suggestion now.